tests/ModelContextProtocol.Tests/Client/McpClientExtensionsTests.cs

```diff
@@ -230,21 +231,24 @@ public async ValueTask DisposeAsync()
         _cts.Dispose();
     }
 
-    private async Task<IMcpClient> CreateMcpClientForServer()
+    private async Task<IMcpClient> CreateMcpClientForServer(
+        McpClientOptions? options = null,
+        CancellationToken? cancellationToken = default)
```

Comment:
It's very unusual for CancellationToken parameters to be nullable. Why does it need to be here?